v2/tools/generator/internal/armconversion/convert_to_arm_function_builder.go

```diff
@@ -629,13 +652,26 @@ func (builder *convertToARMBuilder) convertUserAssignedIdentitiesCollection(
 	locals := params.Locals.Clone()
 
 	itemIdent := locals.CreateLocal("ident")
-	keyTypeAst := destinationType.KeyType().AsType(conversionBuilder.CodeGenerationContext)
-	valueTypeAst := destinationType.ValueType().AsType(conversionBuilder.CodeGenerationContext)
+	keyTypeExpr, kerr := destinationType.KeyType().AsTypeExpr(conversionBuilder.CodeGenerationContext)
```

Comment:
why not use `err` on the lhs here, rather than `kerr` and `verr`?

I thought linting/etc allowed that as long as the other variable in the tuple was new.